### DISCUSSION TITLE
Recommend stop recommending new users to ask on Reddit

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9756] [General Discussions]
Reddit is the best place to get fast help but under current mod settings if you ask more than 1 question a day on the Godot subreddit you get instantly shadow banned until you "interact" with that community an unknown amount of times.  Shadow banned means you are never told you are banned. Interaction means leaving comments on in-dev game screenshots since new users can't give advice at their beginner stage.

For new users, this can be disheartening. Asking a second question, but being shadow banned, getting no replies and not knowing why. Then asking more and more questions and not knowing what to do while you get no replies. This makes the Godot community feel cold and distant, when actually it is warm and friendly.

For this reason:

PROPOSAL: Across all sites Godot should only ever be recommending the actual godot forums for users to get help and never reddit. Any one who wants to help should also be pointed there over Reddit.

**Comment by AThousandShips** [comment]
Who has been recommending Reddit? Was this a member of the team? I've never seen anyone recommend this, we can't tell other people what to do or recommend

**Reply by tiloc** [reply]
It is listed on godotengine.org under Official Communities.

**Reply by AThousandShips** [reply]
Yes but that's a list of many ones, you pick, and the forums are at the top, there's nothing wrong with the Reddit forums, they're a different focus and allows different communication than the forums, just like discord etc.

But people are free to pick the platform they prefer

**Reply by shak2** [reply]
The shadow bans for new users covered above are horrible and disheartening. How many have given up after receiving this treatment?

Anti everything Godot stands for (not warm and friendly).

Minimum - it should be bumped to the bottom of the list.

**Reply by AThousandShips** [reply]
That's specific to new users to handle spam, it's not a reason to abandon an entire forum, just engage some and the ban will go away, or just use the forums, if people pick Reddit then that's up to them, we mainly recommend or direct to the forums, that's the priority

I don't see that the fact that there's interaction limits to prevent abuse should make us abandon an entire forum with a lot of users

Should we also remove the Facebook page because people need to apply and be accepted?

**Reply by shak2** [reply]
It is a shadow ban (not a regular ban). The users do not know they are banned and keep asking questions. The trigger is 2 or more questions in 24 hours. 

Which beginner has not had 2 or more questions a day while working on their dream game?

New users wouldn't know to engage more because it is a shadow ban, they are not informed, you just don't see them, no one does, there posts show to themselves when logged in and no one else. It looks to them like the community is shunning them.

It is evil.

Imagine making your first game at 14 and facing that?

**Reply by AThousandShips** [reply]
I'm sorry you had that experience, but that's probably somewhere in the terms or instructions 

Bur regardless:
* The main suggestion is and has been the forums, not Reddit, people choosing Reddit are largely those who already use it, most people turn to the forums
* There's a big Reddit community of Godot users that shouldn't be killed off

So if this isn't communicated elsewhere, on the Reddit, the solution would be to communicate that instead of just abandoning a flourishing community

Edit: I don't use Reddit but doing some research it sounds like this:
* https://support.reddithelp.com/hc/en-us/articles/204511829-What-is-karma
* https://support.reddithelp.com/hc/en-us/articles/360045989712-Why-can-t-I-see-my-post

So this isn't secret but something you can read about as a new user of Reddit
